hooks/publish.py

```diff
@@ -556,10 +562,10 @@ def undo(self, task_settings, item):
         publish_symlink_path = item.properties.get("publish_symlink_path")
 
         if publish_symlink_path:
-            self.delete_files(task_settings, item, publish_symlink_path)
+            self.delete_files(publish_symlink_path)
```

Comment:
Please do not change this public method signature.

Reply:
I don't understand why. It is not used elsewhere and the decision of what to delete (based on item and task settings) is handled in undo.
Either we should shift that responsibility here, or if this is just a utility function, move it to util or something (make it protected). Not just maintain signatures.